index.html

```diff
@@ -8,11 +8,143 @@
     <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
   </head>
   <body>
-    <nav class="navbar">Watcher</nav>
-    <div class="video">Video</div>
+    <nav class="navbar">
+      &nbsp; &nbsp; Watcher
+      <ul>
+        <li><a href="#">Home</a></li>
+        <li><a href="#">News</a></li>
+        <li><a href="#">Contact</a></li>
+        <li><a href="#">About</a></li>
+      </ul>
+    </nav>
+    <div class="video">
+      <iframe
+        width="100%"
+        height="100%"
+        src="https://www.youtube.com/embed/tgbNymZ7vqY"
+      >
+      </iframe>
+    </div>
+    <div class="description">
+      <b>Bohemian Rhapsody | Muppet Music Video | The Muppets</b><br />
+      104.560 Views
+      <span
+        class="likebtn-wrapper"
+        data-theme="drop"
+        data-identifier="item_1"
+      ></span>
+      <img src="img/share.png" width="20px" alt="share" />
+      <span>&nbsp;SHARE&nbsp;&nbsp;&nbsp;</span>
+      <img src="img/save.png" width="20px" alt="share" />
+      <span>&nbsp;Save</span>
+    </div>
     <div class="lower">
-      <div class="other-videos">Other videos</div>
-      <div class="comments">Comments</div>
+      <div class="other-videos">
```

Comment:
Other videos should be on the right side of the main video, please move them accordingly.
Comments should be underneath the video.